test/configure.in

```diff
@@ -15,7 +15,7 @@ AC_C_CONST
 
 dnl Figure out which math library to use
 case "$host" in
-    *-*-cygwin* | *-*-mingw32*)
+    *-*-cygwin* | *-*-mingw32* | *-windows*)
```

Comment:
This might be incorrect. autoconf supports a windows platform, but I'm not sure if I'm matching it correctly